spec/models/info_request_spec.rb

```diff
@@ -1296,6 +1296,13 @@
       info_request.destroy
       expect(UserInfoRequestSentAlert.where(:info_request_id => info_request.id)).to be_empty
     end
+
+    it 'destroys associated embargoes' do
+      FactoryGirl.create(:embargo, info_request: info_request)
+      info_request.destroy
```

Comment:
This isn't actually testing the record is removed from the DB. Could rewrite as:

`expect { info_request.destroy }.to change(AlaveteliPro::Embargo, :count).by(-1)`